### ISSUE TITLE
Stability

### ISSUE THREAD

**Issue by saenzramiro** [katranci/github-npm-stats#10]
Seems like is not working ok in some repos. For example:

> https://github.com/electron/electron
> https://github.com/electron-userland/electron-builder

I couldn't find the difference, but maybe you can find it.

BTW, great tool. 😃

**Comment by katranci** [comment]
Thanks for reporting this issue @saenzramiro 😃 

tl;dr: Repositories where root package.json file is marked as private are not handled for now.

Some projects have a dedicated build & publish pipeline where package.json is generated automatically. For these projects the package.json in the root folder is merely used for development purposes and the name listed in it may differ from the actual npm package name. Most of these type of projects mark the root package.json file as private. The projects you listed fall into this kind.

For example the electron project builds the package.json in https://github.com/electron/electron/blob/master/script/publish-to-npm.js
Name of the package is copied from the root package.json for this project but there is no guarantee that is the same for all other projects. React project is a good example to this. The npm package name is react but the [project's root package.json lists name as react-build](https://github.com/facebook/react/blob/master/package.json).

I'll add features to improve the stability but there aren't any "quick fixes" for now.

**Comment by saenzramiro** [comment]
Oh I see... so thank you for your time.
